fellowship.js

```diff
@@ -19,30 +19,63 @@ var fellowship = {
   makeMiddleEarth: function(lands) {
     console.log(lands)
     // create a section tag with an id of middle-earth
+    var middleEarth = document.createElement('section');
+      middleEarth.setAttribute('id', 'middle-earth');
     // add each land as an article tag
+    for (var i = 0; i < fellowship.lands.length; i++) {
+    console.log(fellowship.lands[i]);
```

Comment:
indent code two spaces inside curly braces to increase readability and ease debugging.